tensorflow/core/kernels/data/name_utils.h

```diff
@@ -42,6 +42,11 @@ struct DatasetDebugStringParams {
   std::vector<string> args;
 };
 
+struct IteratorPrefixParams {
+  int op_version = 1;
+  string prefix = "";
```

Comment:
this should only contain optional parameters ... in particular, since `prefix` is always required, it should be an explicit parameter (as opposed to part of `IteratorPrefixParams`)